castle/cms/profiles/default/controlpanel.xml

```diff
@@ -105,4 +112,7 @@
   <permission>Plone Site Setup: Users and Groups</permission>
  </configlet>
 
+
+
+
```

Comment:
no need for these extra lines; it's good to avoid making source code changes that aren't necessary